kolibri/core/templates/kolibri/base.html

```diff
@@ -45,6 +46,10 @@
   </script>
 </rootvue>
 {% block frontend_assets %}
+<script>
+  // Set global language variable.
+  var languageCode = "{{ LANGUAGE_CODE }}";
```

Comment:
If it doesn't exist, the code won't break, but if we wanted to test it, we could add it to the test environment.